questionpy_server/settings.py

```diff
@@ -44,6 +46,8 @@ def max_package_size_bigger_then_predefined_value(cls, value: ByteSize) -> ByteS
 
 
 class WorkerSettings(BaseModel):
+    type: Union[str, Type[Worker]] = SubprocessWorker
```

Comment:
Statt `type: Union[str, Type[Worker]]` könnte hier `type: Type[Worker]` stehen, wenn die `_load_worker_class()`-Logik in einen Validator verschoben wird - ist vielleicht etwas cleaner und passt mMn auch gut dort rein.